cmd/internal/config/config.go

```diff
@@ -1,11 +1,16 @@
 // MIT License. Copyright (c) 2020 CQFN
 // https://github.com/cqfn/degitx/blob/master/LICENSE
 
-package main
+// Package config contains config parser code that is shared between front- and back-end parts.
+package config
```

Comment:
@stepanvalyavskiy the package path should be `internal/config` not `cmd/internal` (cmd is only for entry points, not dependencies)